crawler/rabbit/basic_getter.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import os
+import ssl
+from dataclasses import dataclass
+from typing import Optional
+
+from pika import BlockingConnection, ConnectionParameters, PlainCredentials, SSLOptions
+from pika.adapters.blocking_connection import BlockingChannel
+
+from crawler.constants import LOGGER_NAME_RABBIT_MESSAGES
+from crawler.types import RabbitServerDetails
+
+LOGGER = logging.getLogger(__name__)
+MESSAGE_LOGGER = logging.getLogger(LOGGER_NAME_RABBIT_MESSAGES)
+
+
+@dataclass
+class FetchedMessage:
+    headers: dict
+    body: bytes
+
+
+class BasicGetter:
```

Comment:
Is this a candidate for lib-lab-share?

Reply:
It might be.  I did consider this as I was writing it, but I don't think it's a usual use case to pull just one message from the queue and stop consuming.  We want to here so that we can only read the queue for the minimum amount of time, but usually you'd want to use an AsyncConsumer.